apic_ml2/neutron/services/l3_router/l3_apic.py

```diff
@@ -13,6 +13,7 @@
 #    License for the specific language governing permissions and limitations
```

Comment:
I agree the error msg is not too user friendly but I'm just following the convention here. Same thing say if user wants to delete a router which is still connected to an interface or GW port, he will get the same error msg. Actually most of the neutron resources follow this deletion pattern.